docs/pages/application-access/getting-started.mdx

```diff
@@ -23,42 +41,47 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
```

Comment:
I don't think there's enough information in this bullet. A subdomain of which domain? What does "each application" refer to? And what does it mean to have a subdomain?